universal-application-tool-0.0.1/test/support/ResourceCreator.java

```diff
@@ -60,12 +57,10 @@ public QuestionDefinition insertQuestionDefinition() {
   }
 
   public Program insertProgram(String name) {
-    Program program = new Program(name, "description");
-    program.save();
-    return program;
+    return ProgramBuilder.newProgram(name, "description").build();
   }
 
-  public Program insertProgram(String name, BlockDefinition block) {
+  public Program insertPrograms(String name, BlockDefinition block) {
```

Comment:
question: why plural if it only inserts one program?

Reply:
oops! it is also not being used anymore so i deleted it.